pkg/terminal/terminal.go

```diff
@@ -101,6 +101,7 @@ func New(client service.Client, conf *config.Config) *Term {
 		cmds:   cmds,
 		stdout: os.Stdout,
 	}
+	t.line.SetCtrlZStop(true)
```

Comment:
given that this fork of liner exists solely for this feature why not just have it always behave as if ctrlZStop == true?

Reply:
🤔 Good point. I suppose the only argument to keep it this way would be if others rely on this fork, but I don't feel strongly about that. I'll likely change this on Monday.

Reply:
I assumed this was for consistency with `SetCtrlCAborts`. If we follow their convention then the owners of liner might reconsider integrating this into the main repo?

Reply:
This feature was already rejected upstream.

Reply:
I saw that. But you never what the future holds. Hope dies last, right? :0
I think this is fine either way.

Reply:
Well, we aren't scuplting this in granite either.